### PR TITLE
Clickatell Channel type

### PR DESCRIPTION
**Pull request by nicpottier**
Adds a Clickatell channel type.

### CODE REVIEW

**Comment by nicpottier** [comment]
plz review when someone has a minute.

File: temba/api/views.py [hunk 1]
```diff
@@ -3276,6 +3276,107 @@ def post(self, request, *args, **kwargs):
...
+                              '004': DELIVERED,   # received by handset
+                              '005': FAILED,      # error in message
+                              '006': FAILED,      # terminated by user
+                              '007': FAILED,      # error delivering
+                              '008': WIRED,       # msg received
+                              '009': FAILED,      # error routing
+                              '010': FAILED,      # expired
+                              '011': WIRED,       # delayed but queued
+                              '012': FAILED,      # out of credit
+                              '014': FAILED}      # too long
```
Comment (ericnewcomer) [inline temba/api/views.py]: We should probably be capturing and logging out of credit errors for end points that report it.

File: temba/api/views.py [hunk 2]
```diff
@@ -3276,6 +3276,107 @@ def post(self, request, *args, **kwargs):
...
+
+            # check our status
+            status_code = self.request.REQUEST['status']
+            status = STATUS_CHOICES.get(status_code, None)
```
Comment (ericnewcomer) [inline temba/api/views.py]: Maybe unknown status should be failed instead of wired?

**Comment by ericnewcomer** [comment]
Yarp.

**Pull request merged**